### ISSUE TITLE
Jodit Desktop Windows 

### ISSUE THREAD

**Issue by lixaotec** [xdan/jodit#179]
Hey there folks.

Was wondering if would Jodit be feasable to be embedded in a desktop application for windows?

How could that be accomplished?

It would be nice to have it like a replace for notepad, and perhaps MS Word (rich text)

Thanks

**Comment by xdan** [comment]
Yes — Jodit runs in any Chromium-based desktop shell. Wrap it in **Electron** (or Tauri) and load a page that initializes `Jodit.make('#editor')`; for file/notepad-like behavior, hook the editor's value into the host's file APIs via the preload bridge. There's also a community Jodit desktop discussion, but the standard path is Electron + a normal Jodit web build. Closing as answered.
